src/data/option/basic.lean

```diff
@@ -142,4 +142,25 @@ theorem lift_or_get_choice {f : α → α → α} (h : ∀ a b, f a b = a ∨ f
 | none     (some b) := or.inr rfl
 | (some a) (some b) := by simpa [lift_or_get] using h a b
 
+/-! Add functorial coercions and basic lemmas about it. -/
```

Comment:
```suggestion
/-- Add functorial coercions and basic lemmas about it. -/
```
The `/-! -/` style is for a module-level docstring, not for individual statements.

Reply:
Oh, I wanted to add a comment for the section and `/--` does not work for that (`command does not accept doc string`) so I thought `/-!` should be used here. Should I change it to `/-`?

Reply:
Yes, I think so. But @robertylewis is the documentation expert atm.

Reply:
This is had to answer without a use case for the module doc strings. My instinct is that it should be `/-` for now. I'd guess that if/when we have a first pass at a documentation generator, it will be easier to look for a single module doc string per file, rather than try to distinguish the header from smaller comments like this.

Reply:
Alright - changed it.